fulltext/backends/__csv.py

```diff
@@ -10,7 +10,13 @@
 
 def readlines(f):
     for line in f.readlines():
-        yield line.decode('utf8')
+        # Decoding may be necessary.
+        try:
+            line = line.decode('utf8')
+        except AttributeError:
+            pass
```

Comment:
isn't this unnecessary now that we make sure file is binary up-front?

Reply:
The `try...except` is unecessary.